### ISSUE TITLE
Standard test set failed due to http test failure

### ISSUE THREAD

**Issue by lbergelson** [samtools/htsjdk#1287]
I saw a test failure in the main test build caused by failing SRA tests.  It seems we're accidentally running the SRA tests as part of the standard test build again.

```
[32mSRAAccessionTest:[0m
[32mHttpUtilsTest:[0m
[32m- testGetETag(http://broadinstitute.github.io/picard/testdata/index_test.bam)[0m
[31m- testGetETag(http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/current.tree) *** FAILED ***[0m
[31m  java.lang.AssertionError: expected object to not be null[0m
[31m  at org.testng.Assert.fail(Assert.java:96)[0m
[31m  at org.testng.Assert.assertNotNull(Assert.java:686)[0m
[31m  at org.testng.Assert.assertNotNull(Assert.java:671)[0m
[31m  at [secure].samtools.util.HttpUtilsTest.testGetETag(HttpUtilsTest.java:31)[0m
[31m  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[31m  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)[0m
[31m  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[0m
[31m  at java.lang.reflect.Method.invoke(Method.java:498)[0m
[31m  at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)[0m
[31m  at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)[0m
[31m  ...[0m
[32m- testGetHeaderField(http://broadinstitute.github.io/picard/testdata/index_test.bam)[0m
[31m- testGetHeaderField(http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/current.tree) *** FAILED ***[0m
[31m  java.lang.AssertionError: expected object to not be null[0m
[31m  at org.testng.Assert.fail(Assert.java:96)[0m
[31m  at org.testng.Assert.assertNotNull(Assert.java:686)[0m
[31m  at org.testng.Assert.assertNotNull(Assert.java:671)[0m
[31m  at [secure].samtools.util.HttpUtilsTest.testGetHeaderField(HttpUtilsTest.java:23)[0m
[31m  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[31m  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)[0m
[31m  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[0m
[31m  at java.lang.reflect.Method.invoke(Method.java:498)[0m
[31m  at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)[0m
[31m  at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)[0m
[31m  ...[0m

===============================================
Command line suite
Total tests run: 4, Failures: 2, Skips: 0
===============================================
```

**Comment by lbergelson** [comment]
I miss read this failure.  It wasn't the SRA test failing, it was the http test right after it.

**Comment by lbergelson** [comment]
closed by #1289
